docs/anomaly-detection/docs/README.md

```diff
@@ -0,0 +1,27 @@
+---
+sort: 1
+title: Docs
+weight: 1
+menu:
+  docs:
+    identifier: vmanomaly-docs
+    parent: "anomaly-detection"
+    weight: 1
+    # sort: 1
+aliases:
+  - /anomaly-detection/docs/
+  - /anomaly-detection/docs/index.html
+---
+
+# Docs
+
+This chapter describes different sections of a config to launch VictoriaMetrics Anomaly Detection (or simply [`vmanomaly`](/vmanomaly.html)) service:
+
+- [Model(s) section](models/README.md) - Required
```

Comment:
Sorting here is different. Here in the list Models are coming first. But when clicking on "Next" it shows Reader page. Here the order should be like in the menu section

Reply:
And information about licensing from vmanomaly section should be moved somewhere here

Reply:
mentioned in main page (orange "VM anomaly detection" page `/anomaly-detection`), added references to existing pages and sections